### ISSUE TITLE
the validation set and test set in paper

### ISSUE THREAD

**Issue by JamesLee789** [JXingZhao/ContrastPrior#7]
Hi. I'm afraid that I only found the training set in your opensource project. Could you please opensource the validation set and the test set? As you mentioned in the paper, "We also sample 100 images from NJU2000 and 50 from NLPR as the validation set. The rest images are for testing." Thanks a lot!
